scripts/pkg/build_templates/opensearch/deb/debian/postinst

```diff
@@ -57,17 +57,30 @@ if command -v systemd-tmpfiles > /dev/null; then
     systemd-tmpfiles --create opensearch.conf
 fi
 
-# Messages
-echo "### NOT starting on installation, please execute the following statements to configure opensearch service to start automatically using systemd"
-echo " sudo systemctl daemon-reload"
-echo " sudo systemctl enable opensearch.service"
-echo "### You can start opensearch service by executing"
-echo " sudo systemctl start opensearch.service"
+# Restart the services after an upgrade
+if [ "$1" == "configure" ] && [ "$2" != "" ]; then
+    if command -v systemctl >/dev/null && systemctl is-enabled opensearch.service >/dev/null; then
+        echo "Restarting opensearch.service after upgrade"
+        systemctl start opensearch.service
+    fi
+    if command -v systemctl >/dev/null && systemctl is-enabled opensearch-performance-analyzer.service >/dev/null; then
+        echo "Restarting opensearch-performance-analyzer.service after upgrade"
+        systemctl start opensearch-performance-analyzer.service
+    fi
+else
+    # Messages
+    echo "### NOT starting on installation, please execute the following statements to configure opensearch service to start automatically using systemd"
+    echo " sudo systemctl daemon-reload"
+    echo " sudo systemctl enable opensearch.service"
+    echo "### You can start opensearch service by executing"
+    echo " sudo systemctl start opensearch.service"
 
-if [ -d ${product_dir}/plugins/opensearch-security ]; then
-    echo "### Create opensearch demo certificates in ${config_dir}/"
-    echo " See demo certs creation log in ${log_dir}/install_demo_configuration.log"
+    if [ -d ${product_dir}/plugins/opensearch-security ]; then
+        echo "### Create opensearch demo certificates in ${config_dir}/"
+        echo " See demo certs creation log in ${log_dir}/install_demo_configuration.log"
+    fi
```

Comment:
Let us move this part outside of the if block, so it will always run on installation/reinstallation/upgrade.

Reply:
Sounds good, I only put it in the if-block because it looked like information that seemed like it was only relevant on the initial installation. If its preferred to also have this text show up on upgrades/reinstallation that sounds good to me 👍

Reply:
Well, I think I misread the line as the actual script is running on line 24.
We can keep this unchanged for now.

Sorry about it.